tools/scylla-nodetool.cc

```diff
@@ -109,6 +109,38 @@ struct fmt::formatter<file_size_printer> : fmt::formatter<string_view> {
     }
 };
 
+struct history_entry {
+    struct sstable {
+        std::string generation;
+        std::string origin;
+        int64_t size;
+    };
+
+    utils::UUID id;
+    shard_id shard_id;
+    std::string table;
+    std::string keyspace;
+    std::string compaction_type;
+    int64_t compacted_at;
+    int64_t bytes_in;
+    int64_t bytes_out;
+    std::map<int32_t, int64_t> rows_merged;
+    std::vector<sstable> sstables_in;
+    std::vector<sstable> sstables_out;
+    int64_t total_tombstone_purge_attempt;
+    int64_t total_tombstone_purge_failure_due_to_overlapping_with_memtable;
+    int64_t total_tombstone_purge_failure_due_to_overlapping_with_uncompacting_sstable;
+};
```

Comment:
Can't we reuse the type used in Scylla core? You can include any header here.

Reply:
Reused